### DISCUSSION TITLE
Join us on Slack!  🎉

### DISCUSSION THREAD

**Discussion by mitrotasios** [tremorlabs/tremor-npm#428] [Announcements]
We created another channel where you can complain about the lack of dark mode support 😜 Join our Slack Community to help shape the future of Tremor:

👉 Invite Link: [https://join.slack.com/t/tremor-community/shared_invite/zt-1u8jqmcmq-Fdr9B6MbnO7u8FkGh~2Ylg](https://t.co/g99pltTvIC)

**Comment by kempe-codelabs** [comment]
It's not possible to join this one if you don't have an @tremor.so email.

"The email address must match one of the domains listed below. Please try another email."

**Reply by christopherkindl** [reply]
Hi @kempe-codelabs, thanks for the bug. Should be fixed now, try: tremor.so/slack
